com.chartboost.mediation/Runtime/FullScreen/Interstitial/IChartboostMediationInterstitialEvents.cs

```diff
@@ -5,7 +5,6 @@ namespace Chartboost.FullScreen.Interstitial
     /// <summary>
     /// All interstitial placement callbacks.
     /// </summary>
-    [Obsolete("IChartboostMediationInterstitialEvents has been deprecated, use the new fullscreen API instead.")]
```

Comment:
So... did this API comeback to life? 👀

Reply:
Intentional, we only deprecate its functionality, if we specify the interface too, we need to deprecate all classes implementing it, which is not ideal since only the functionality is deprecated.